aten/src/ATen/ScalarType.h

```diff
@@ -143,5 +143,6 @@ static inline ScalarType promoteTypes(ScalarType a, ScalarType b) {
 struct Tensor;
 typedef ArrayRef<int64_t> IntList;
 typedef ArrayRef<Tensor> TensorList;
+typedef std::string String;
```

Comment:
Can you please remove this typedef? You should be able to use `std::string` in every place where you used `String`